feature-libs/customer-ticketing/components/customer-ticketing-dialog/default-customer-ticketing-form-layout.config.ts

```diff
@@ -1,9 +1,9 @@
 import { DIALOG_TYPE, LayoutConfig } from '@spartacus/storefront';
-import { CustomerTicketingReopenDialogComponent } from './customer-ticketing-reopen-dialog.component';
+import { CustomerTicketingReopenDialogComponent } from '../customer-ticketing-reopen';
 
 export const defaultCustomerTicketingFormLayoutConfig: LayoutConfig = {
```

Comment:
instead of putting this 'dialog' within the shared dialog component, you can just place it in the individual folders to come like the create, reopen, and close as part of it's small 'domain'

Reply:
@Parthlakhani I think this was missed

the default 'layout' config would be placed in the /customer-ticketing-reopen folder, and you would rename the file as `default-customer-ticketing-reopen-layout.config.ts`

Reply:
basically, each of these dialog will have it's own layout config within it's of its own folder